pages/signup.css

```diff
@@ -0,0 +1,289 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+
+}
+.container{
+    max-width: 1440px;
+    max-width: 100vw;
+    margin: 0 auto;
+    height: 800px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+#logo{
+    margin-left: -20px;
+    margin-bottom: 8px;
+}
+
+.component-1{
+    width: 45%;
+    height: 800px;
+    background-color: #fff;
+    padding:25px 30px 25px 60px;
+}
+
+.component-2{
+    width: 55%;
+    height: 800px;
+    background-color:#D0EBFF;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.component-2 img{
+    margin-top: 60px;
+}
+
+.form{
+    display: flex;
+    flex-direction: column;
+}
+
+.component-1 h2{
+    font-size: 36px;
+    font-weight: 500;
+    color: #1864AB;
+
+}
+
+.component-1 h4{
+    font-size: 20px;
+    font-weight: 400;
+    color: #595F69;
+}
+
+.component-1 label{
+    font-size: 14px;
+    font-weight: 400;
+    color: #8A9099;
+    margin-top: 10px;
+}
+
+.form input{
+    width: 420px;
+    height: 40px;
+    border-radius: 10px;
+    font-size: 14px;
+    padding: 5px 10px;
+    margin-top: 5px;
+    border: 1px solid #8A9099;
+    color: #3F434A;
+    font-weight: 500;
+}
+
+.position{
+    width: 420px;
+    height: 15px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 16px;
+}
+.position label{
+    margin-top: 0;
+    margin-right: 8px;
+    font-size: 14px;
+    color: #1864AB;
+    font-weight: 400;
+}
+
+.position input{
+    width: 16px;
+    height: 16px;
+    margin-right: 5px;
+}
+
+.terms-condition{
+    display: flex;
+    font-size: 14px;
+    font-weight: 400;
+    color: #595F69;
+    margin-top: 16px;
+}
+.terms-condition input{
+    margin-right: 8px;
+    width: 16px;
+    height: 16px;
+}
+
+.terms-condition a{
+    text-decoration: none;
+    color: #1864AB;
+}
+
+.component-1 button{
+    background-color: #1864AB;
+    width: 420px;
+    height: 40px;
+    font-family: 'Poppins', sans-serif;
+    color: #fff;
+    font-size: 15px;
+    font-weight: 500;
+    border-radius: 10px;
+    border: none;
+    margin-top: 12px;
+    cursor: pointer;
+}
+.sign-up{
+    width: 420px;
+    height: 15px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16px 0;  
+}
+.sign-up p{
+    text-transform: uppercase;
+    color: #8A9099;
+    font-size: 12px;
+    font-weight: 400;
+    opacity: 0.6;
+    margin: 0 5px;
+}
+.sign-up hr{
+    width: 150px;
+    opacity: 0.6;
+
+}
+
+.social-media{
+    width: 420px;
+    display: flex;
+    flex-direction: column;
+    vertical-align: middle;
+
+}
+.social-media a{
+    /* padding: 8px 30px; */
```

Comment:
Remove commented lines

Reply:
Thank you, @supuna97, for your valuable comments. I will make the edits you mentioned. 😊